python/fapolicy_analyzer/ui/state_manager.py

```diff
@@ -73,5 +73,16 @@ def __update_dirty_queue(self):
             self.changeset_queue_updated()
         return self.bDirtyQ
 
+    def get_path_action_list(self):
+        listReturn = []
+
+        # Iterate through the StateManagers Changeset list
+        for e in self.listChangeset:
+            # Each changeset contains a dict with at least one Path/Action pair
+            for t in e.get_path_action_map().items():
+                listReturn.append(t)
+
+        return listReturn
```

Comment:
This whole think can be done with python comprehensions 
```suggestion
        return [t for e in self.listChangeset for t in e.get_path_action_map().items]
```

Reply:
Done